src/Resources/skeleton/crud/controller/Controller.tpl.php

```diff
@@ -0,0 +1,104 @@
+<?= "<?php\n"; ?>
+
+namespace <?= $namespace; ?>;
+
+use <?= $entity_full_class_name; ?>;
+use <?= $form_full_class_name; ?>;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Route("<?= $route_path; ?>", name="<?= $route_name; ?>_")
```

Comment:
Hmm, putting the `name=` here is pretty clever... but I personally would rather put the *full* route names below above each method. So, I like the path *prefix* up here, but not the name prefix.

Reply:
agreed 👍